### PR TITLE
chore: add useful npm scripts

### PR DESCRIPTION
**Pull request by grant**
- Adds some useful scripts to this package: testing an individual test, tsc watching

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,10 +16,12 @@
   },
   "scripts": {
     "test": "mocha build/test",
+    "test-individual": "mocha build/test -g 'loading function'",
```
Comment (matthewrobertson) [inline package.json:19]: Did you mean to hardcode the `'loading function'` here?

Comment (matthewrobertson) [reply package.json:19]: FYI this is pretty easy to do today with `npm run test -- -g 'my test'` so I am not sure it is worth adding a script for this

**Review by matthewrobertson — APPROVED** [review]
See comment about `test-individual`, otherwise LGTM

File: package.json (hunk 1 above)
Comment (grant) [reply package.json]: Sounds good. I wanted to document how to do this somewhere.

I removed the script but added a note on how to do this in the contributing section.

**Pull request merged**